<launch>
    <!-- 设置节点参数 -->
    <arg name="camera_topic" default="/mipi_cam/image_raw" />
    <!-- <arg name="camera_info_topic" default="/usb_cam/camera_info" /> -->

    <!-- 启动 image_subscriber 节点 -->
    <node name="camera_calibration" pkg="camera_calibration" type="camera_calibration_node" output="screen"  launch-prefix="gdb -ex run --args">
        <param name="camera_topic" value="$(arg camera_topic)" />
        <!-- <param name="camera_info_topic" value="$(arg camera_info_topic)" /> -->
    </node>
</launch>
